lib/kvs/kvs_rparams.c

```diff
@@ -208,7 +208,7 @@ static const struct param_spec pspecs[] = {
         .ps_stringify = param_default_stringify,
         .ps_jsonify = param_default_jsonify,
         .ps_default_value = {
-            .as_uscalar = 19,
+            .as_uscalar = 23,
```

Comment:
Increase the split size to help reduce tiny spills, seems most effective during load before too many splits have occurred.  Works in concert with the increased root spill size.